modules/storage-expanding-local-volumes.adoc

```diff
@@ -0,0 +1,21 @@
+// Module included in the following assemblies
+//
+// * storage/expanding-persistent-volumes.adoc
+
+:_content-type: PROCEDURE
+[id="expanding-local-volumes{context}"]
```

Comment:
Should this be (add the underscore before context):
[id="expanding-local-volumes_{context}"]